components/suggest/src/db.rs

```diff
@@ -974,6 +1097,11 @@ impl<'a> SuggestDao<'a> {
                     wiki_insert.execute(suggestion_id, wikipedia)?;
                 }
             }
+            fts_insert.execute(
+                suggestion_id,
+                &common_details.full_keywords_joined(),
+                &common_details.title,
+            )?;
```

Comment:
Sorry, just thought of something: Should this be gated on provider constraints in the ingest constraints? Most people won't be in the experiment, and ideally they wouldn't pay the cost of ingesting into the FTS table. Desktop can easily pass in some ingest constraints for users in the experiment. Maybe it's not a big deal though.

Reply:
My concern with that is that we won't re-ingest records if they've already been ingested, and I couldn't figure out a good way to handle this in `should_reprocess_record` with the data we currently have.

It does add some time to ingestion, but not all that much.  I put some metrics in the title.

Maybe we could store the provider constraints in a column in `ingested_records`, but that seems more like a long-term improvement to me.

Reply:
Seems doable: If the constraints are `FtsAgainstFullKeywords` or `FtsAgainstTitle` and the FTS table is empty, then reprocess? I'm assuming it's simple to tell whether the table is empty like it is for non-virtual/non-FTS tables. I don't want to block on that though, and thanks for doing the benchmarks.

> Maybe we could store the provider constraints in a column in `ingested_records`, but that seems more like a long-term improvement to me.

I had that thought too when I added `should_reprocess_record`. That would be a general, sure-fire way of reingesting, so it might be worth pursuing.

Reply:
Good point, I think we can do it.  My last push added this.